frontend/src/sass/components/_footer.scss

```diff
@@ -1,4 +1,4 @@
-.footer {
+.global-footer {
 	background: $lighter-black-colour;
 	
 	&__list {
```

Comment:
you mean 2 spaces instead of 4? We have been using 4 throughout but just the 3rd party code is 2.